geode-modules/build.gradle

```diff
@@ -163,5 +162,7 @@ dependencies {
     module4WithManifestCompileOnly(sourceSets.test.output)
     module5WithManifestCompileOnly(sourceSets.test.output)
 
-    module1WithManifestCompile('org.springframework:spring-core')
+    module1WithManifestCompile('com.google.guava:guava:29.0-jre')
```

Comment:
Check in `buildSrc/src/main/groovy/org/apache/geode/gradle/plugins/DependencyConstraints.groovy` to see if this version is already defined